src/features/pastelStatistics/components/chart/index.ts

```diff
@@ -0,0 +1 @@
+export * from './EChartsLineChart'
```

Comment:
We don't need this file. If you need this component anywhere outside this feature. You can export it from the index file of the feature